apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -196,6 +196,18 @@ export NEW_RELIC_API_NAME ?= "kuma-backend-${NEW_RELIC_MONITOR_SUFFIX};${NEW_REL
 export NEW_RELIC_KUMASCRIPT_NAME ?= "kumascript-${NEW_RELIC_MONITOR_SUFFIX}"
 export NEW_RELIC_CELERY_NAME ?= "kuma-backend-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
 
+export DATADOG_REDIS_DEPLOYMENT_NAME ?= mdn-dd-redis
+export DATADOG_SECRETS_NAME ?= datadog-secrets
+
+export DATADOG_API_KEY ?= foo
+export DATADOG_API_KEY_BASE64 ?= $(shell echo -n "${DATADOG_API_KEY}" | base64)
+
+export DATADOG_REDIS_CONFIG ?= foo
+# note the base64 -w0, this prevents base64 from splitting the string w/ newlines
+# see https://kubernetes.io/docs/concepts/configuration/secret/
+export DATADOG_REDIS_CONFIG_BASE64 ?= $(shell echo -n "${DATADOG_REDIS_CONFIG}" | base64 -w0)
```

Comment:
@metadave @jwhitlock The `base64` command on my Mac complains about the `-w0` option. It appears that `base64` on Mac is different than on Linux.

Reply:
I've noted that in the doc: https://github.com/mozmeao/infra/pull/561/files#diff-04209630fe5af9be901ec0af15efc1adR28

Reply:
Also, it appears that the osx version of `base64` outputs the correct format by default.